### PR TITLE
BC attorney address add and unit testing

### PR DESCRIPTION
**Pull request by jig-patel**
BC attorney address add and unit testing

### CODE REVIEW

File: frontend/src/organizations/OrganizationEditContainer.js [hunk 1]
```diff
@@ -3,229 +3,272 @@
...
+    this._handleInputChange = this._handleInputChange.bind(this);
+    this._handleCreate = this._handleCreate.bind(this);
+    this._handleUpdate = this._handleUpdate.bind(this);
   }
 
-  componentDidMount () {
-    if (this.props.mode === 'add') {
-      return
+  componentDidMount() {
+    if (this.props.mode === "add") {
```
Comment (AlexZorkin) [inline frontend/src/organizations/OrganizationEditContainer.js:59]: We are using JS Standard linting which doesn't use semi-colons. Try running npm run lint to see any required changes.

File: backend/api/migrations/0204_auto_20230316_2115.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0203_auto_20230312_0823'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```
Comment (AlexZorkin) [inline backend/api/migrations/0204_auto_20230316_2115.py:13]: are we sure there isn't any information we're losing with this removal?

**Review by AlexZorkin — CHANGES_REQUESTED** [review]

File: backend/api/migrations/0204_auto_20230316_2115.py (hunk 2 above)
Comment (jig-patel) [reply backend/api/migrations/0204_auto_20230316_2115.py:13]: We would need to manually update DB to append Address_line_3 data with Address_line_2 and county content with Province unless there is better way to do it. In a local i did update DB manually with query to make sure we don't lose data.

File: frontend/src/organizations/OrganizationEditContainer.js (hunk 1 above)
Comment (jig-patel) [reply frontend/src/organizations/OrganizationEditContainer.js:59]: i did ran "npm run lint" and it didn't complain about semi-colon.

File: backend/api/migrations/0204_auto_20230316_2115.py (hunk 2 above)
Comment (kuanfandevops) [reply backend/api/migrations/0204_auto_20230316_2115.py:13]: @jig-patel please update the based branch to release-2.5.0 as 2.4.0 has been released to prod

Comment (jig-patel) [reply backend/api/migrations/0204_auto_20230316_2115.py:13]: @kuanfandevops thanks. will do

File: frontend/src/organizations/OrganizationEditContainer.js (hunk 1 above)
Comment (jig-patel) [reply frontend/src/organizations/OrganizationEditContainer.js]: i will take that previous statement back. npm run lint fixed semicolon and i have pushed that changes.

**Pull request closed without merging**